maskhit/train.py

```diff
@@ -51,7 +63,8 @@
 else:
     args.outcomes = [args.outcome]
 
-args.patch_spec = f"mag_{args.magnification}-size_{args.patch_size}"
+
+args.patch_spec = f"mag_{str(args.magnification) + '.0'}-size_{args.patch_size}"

```

Comment:
Appending '.0' seems very immature operation with less robustness. how about casting in float, then apply formatting in f-string, using .1f. Technically you are making assumption that magnification is either int or float with at most single decimal point. I think better just casting to float without formatting.